homeassistant/components/scrape/config_flow.py

```diff
@@ -95,35 +95,50 @@
     vol.Optional(CONF_ENCODING, default=DEFAULT_ENCODING): TextSelector(),
 }
 
+NONE_SENTINEL = "none"
+
 SENSOR_SETUP = {
     vol.Required(CONF_SELECT): TextSelector(),
     vol.Optional(CONF_INDEX, default=0): NumberSelector(
         NumberSelectorConfig(min=0, step=1, mode=NumberSelectorMode.BOX)
     ),
     vol.Optional(CONF_ATTRIBUTE): TextSelector(),
     vol.Optional(CONF_VALUE_TEMPLATE): TemplateSelector(),
-    vol.Optional(CONF_DEVICE_CLASS): SelectSelector(
+    # Note: we set default to ensure that frontend does not omit the result
+    vol.Optional(CONF_DEVICE_CLASS, default=NONE_SENTINEL): SelectSelector(
```

Comment:
Is this change needed when the none-sentinel is no longer the empty string?

Reply:
No longer needed - removed.